### ISSUE TITLE
JDK-20 support

### ISSUE THREAD

**Issue by mhjort** [clj-python/libpython-clj#243]
Should this library support JDK-20? With JDK-19 everything works. However, when upgrading to JDK-20 I get following error:

```
Caused by: java.lang.ClassNotFoundException: jdk.incubator.foreign.MemoryAddress
```

**Comment by cnuernber** [comment]
yes it should - jdk-20 has same foreign iface as jdk-19 so I guess the system is loading the wrong bindings.

**Comment by sbenhaim** [comment]
In JDK 20, `jdk.incubator.foreign.MemoryAddress` has been moved to `java.lang.foreign.MemorySegment`.

https://www.jrebel.com/blog/java-20-features

**Comment by cnuernber** [comment]
JDK-20 changed the API again.  I would rather wait for the next long term release than keep updating.  If someone wants to take a crack at it you need to fix it in dtype-next.  [Here](https://github.com/cnuernber/dtype-next/blob/master/src/tech/v3/datatype/ffi/mmodel_jdk19.clj) is a reasonable place to start.
